include/imex/Transforms/FuncUtils.hpp

```diff
@@ -0,0 +1,54 @@
+//===- FuncUtils.hpp -  --------------*- C++ -*-===//
```

Comment:
Can we move this file/content to 'include/internal'? I am not sure if 'internal' is the right term, we can also rename it to Util or whatever else.